examples/pygeosxExamples/hydraulicFractureWithMonitor/hydraulicFractureWithMonitor.py

```diff
@@ -9,27 +10,33 @@
 # Get the MPI rank
 comm = MPI.COMM_WORLD
 rank = comm.Get_rank()
```

Comment:
I believe we already discussed this, but at least finding `rank` in the `if __name__ == '__main__':` part and explicitly giving it to `run_problem` would be cleaner (less side effects).